ToolBox/RandomDataGenerator/RandomDataGeneratorSettings.cs

```diff
@@ -0,0 +1,263 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using QuantConnect.Brokerages;
+using QuantConnect.Data;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.ToolBox.RandomDataGenerator
+{
+    public class RandomDataGeneratorSettings
+    {
+        private static int MarketCode = 100;
+
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+        public SecurityType SecurityType { get; set; } = SecurityType.Equity;
+        public DataDensity DataDensity { get; set; } = DataDensity.Dense;
+        public Resolution Resolution { get; set; } = Resolution.Minute;
+        public string Market { get; set; }
+        public bool IncludeCoarse { get; set; } = true;
+        public int SymbolCount { get; set; }
+        public TickType[] TickTypes { get; set; }
+
+        public static RandomDataGeneratorSettings FromCommandLineArguments(
+            string startDateString,
+            string endDateString,
+            string symbolCountString,
+            string market,
+            string securityTypeString,
+            string resolutionString,
+            string dataDensityString,
+            string includeCoarseString,
+            ConsoleLeveledOutput output
+            )
+        {
+            int symbolCount;
+            bool includeCoarse;
+            TickType[] tickTypes;
+            Resolution resolution;
+            DataDensity dataDensity;
+            SecurityType securityType;
+            DateTime startDate, endDate;
+
+            // --start
+            if (!DateTime.TryParseExact(startDateString, DateFormat.EightCharacter, null, DateTimeStyles.None, out startDate))
+            {
+                output.Error.WriteLine($"Required parameter --start was incorrectly formatted. Please specify in yyyy-MM-dd format. Value provided: '{startDateString}'");
```

Comment:
Inconsistency between the documentation and the functionality. If the date is formatted as `yyyy-MM-dd` it is not parsed.
![imagen](https://user-images.githubusercontent.com/10123175/51189391-bb7e9a00-18be-11e9-8806-a98774180d3e.png)

Reply:
yup, yup, it was changed to be in-sync with the other date format -- yet again --another amazing find, love the thorough review!! I will update the documentation. I'll also check the error validation messages and confirm they have the correct format, if they mention it as well